deploy/README.md

```diff
@@ -1,9 +1,9 @@
-Bringing up a local instance of DBSP
+Bringing up a local instance of Feldera
```

Comment:
This cannot be right, Feldera is a company name.
Must be something else, like "Feldera demo".
This applies to all changes in this file.

Reply:
> This cannot be right, Feldera is a company name. Must be something else, like "Feldera demo". This applies to all changes in this file.

Did you review this https://github.com/feldera/dbsp/commit/b57ffb970a7b675b26b93f27725b366c8535854a?